### PR TITLE
CronTab configuration

### LINKED ISSUE #69: CronTab for WebScraping APIs

### PR DESCRIPTION
**Pull request by RomTamam**
Fix #69 69

### CODE REVIEW

File: cron.out [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (simzacks) [inline cron.out]: this file should be removed and added to your .gitignore

File: setup.sh [hunk 2]
```diff
@@ -16,5 +16,9 @@ pipenv run python manage.py migrate
 
 
 # run our app. Nohup and “&” are used to let the setup script finish
+sudo dnf install crontabs
+sudo systemctl start crond
+python manage.py crontab add
```
Comment (simzacks) [inline setup.sh:19]: looks like these 3 lines should be above the comment on line 18

**Review by simzacks — COMMENTED** [review]
You need to get rid of all the files that are also in PR #97 and keep this PR focused only on the cron stuff

File: cron.out (hunk 1 above)
Comment (RomTamam) [reply cron.out]: done

File: setup.sh (hunk 2 above)
Comment (RomTamam) [reply setup.sh:19]: fixed

**Comment by RomTamam** [comment]
push -f didn't create a new PR for me, can I get a hint?

File: cron.out [hunk 3]
```diff
@@ -0,0 +1,29 @@
+hello
+hello
+hello
```
Comment (Yarboa) [inline cron.out:3]: This file should not be added to git

File: ultimatejobweb/cron.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import os
+
+
+def my_scheduled_job():
+    os.system('echo hello >> /vagrant/cron.out')
```
Comment (Yarboa) [inline ultimatejobweb/cron.py:5]: so this PR has two PRs 
One is the cron, which is not running any script. 
2nd is adding RH and tests, I think you should split PRs.

**Pull request closed without merging**